lib/utils.js

```diff
@@ -147,14 +156,28 @@ const utils = {
   },
 
   saveUserConfig: async (config) => {
-    const conf = yaml.safeDump(config);
     const f = resolveUserConfig();
+    const key = Object.keys(config)[0];
+
     await ensureFile(f);
-    await writeFile(f, conf);
+    const userConfig = yaml.safeLoad(await readFile(f)) || {};
+    userConfig[key] = config[key];
```

Comment:
why find the key and then set it like that when you could just use mergeRight? I would also do this after the update to apiToken to the DEFAULT_URL.

Reply:
The key I'm finding is also used to decide whether to update or delete the old apiToken entry. Since I have it I might as well use it twice, no?

Reply:
```
const userConfig = yaml.safeLoad(await readFile(f)) || {};
if (has('apiToken', userConfig)) {
     const token = userConfig.apiToken;
    userConfig[DEFAULT_URL] = { apiToken: token };
    delete userConfig.apiToken;
}
userConfig = mergeRight(userConfig, config);
const newConfig = yaml.safeDump(userConfig);
await writeFile(f, newConfig);
return f;
```